examples/database-restore/db-restore.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+namespace="bitwarden"
+
+kubectl delete job -n $namespace -l app=bitwarden-restore
```

Comment:
Add this arg to be resilient to first restore: `--ignore-not-found=true`
```suggestion
kubectl delete job -n $namespace --ignore-not-found=true  -l app=bitwarden-restore
```

Reply:
Using the label keeps it from failing.  It just says that no resources were found and keeps going. I can add it if you think it needs to be there, but it is not necessary with the label query.